site/docs/v1/tech/apis/identity-providers/_identity-provider-login-method-request-parameter.adoc

```diff
@@ -4,15 +4,24 @@ The login method to use for this Identity Provider.
 +
 The possible values are:
 +
-* `UsePopup` - When logging in use a popup window and the {idp_display_name} javascript library.
-* `UseRedirect` - When logging in use the {idp_display_name} Oauth redirect login flow.
+ * `UsePopup` - When logging in use a popup window and the {idp_display_name} JavaScript library.
+ * `UseRedirect` - When logging in use the {idp_display_name} OAuth redirect login flow.
+ * `UseVendorJavaScript` - When logging in allow the {idp_display_name} JavaScript library to determine the login method.
+ifeval::["{idp_type}" == "Google"]
++
+[since]#Since 1.44.0#
+The `UsePopup` and `UseVendorJavaScript` methods will render the button in an `iframe` because of changes to the Google JavaScript API.
```

Comment:
I don't know if we need to document how Google does this - in theory it could change. Once it is in Google JavaScript land, it is a black box. 

Maybe we could say, by default, `UsePopup` and `UseVendorJavaScript` will both result in a popup, because that is the default in the Google GIS SDK. 

But then if you want to manually set `ux_mode` while using  `UseVendorJavaScript` we'll do whatever you configure, which could be a redirect to Google. This is still different than `UseRedirect` but the end result is similar.

Reply:
I did not specify the login method here. The button is rendered in an `iframe` either way.

Reply:
I ended up removing this line. The point I was trying to get across is really the same as the comment about button text: the new API does not allow full button customization.

This phrasing is confusing and redundant.